test/framework/test_composite.py

```diff
@@ -1000,3 +1017,127 @@ def circuits(self):
+
+        exp1 = FakeExperiment([0])
+        exp2 = FakeExperiment([1])
+        par_exp1 = ParallelExperiment([exp1, exp2], flatten_results=True)
+
+        exp3 = FakeExperiment([0])
+        exp4 = FakeExperiment([1])
+        par_exp2 = ParallelExperiment([exp3, exp4], flatten_results=True)
+
+        # Set a batch experiment
+        batch_exp = BatchExperiment([par_exp1, par_exp2], flatten_results=True)
+        exp_data = batch_exp.run(backend)
+
+        self.assertExperimentDone(exp_data)
+
+    def test_new_workflow_all_same(self):
+
+        data = [
+            {
+                "metadata": {
+                    "experiment_type": "BatchExperiment",
+                    "composite_metadata": [
+                        {
+                            "experiment_type": "ParallelExperiment",
+                            "composite_index": [0, 1],
+                            "composite_metadata": [{}, {}],
+                            "composite_qubits": [[0], [1]],
+                            "composite_clbits": [[], []],
+                        }
+                    ],
+                    "composite_index": [0],
+                }
+            },
+            {
+                "metadata": {
+                    "experiment_type": "BatchExperiment",
+                    "composite_metadata": [
+                        {
+                            "experiment_type": "ParallelExperiment",
+                            "composite_index": [0, 1],
+                            "composite_metadata": [{}, {}],
+                            "composite_qubits": [[0], [1]],
+                            "composite_clbits": [[], []],
+                        }
+                    ],
+                    "composite_index": [1],
+                },
+            }
+        ]
+        
+        
+        backend = FakeBackend()
+
+        exp1 = FakeExperiment([0])
+        exp2 = FakeExperiment([1])
+        par_exp1 = ParallelExperiment([exp1, exp2], flatten_results=True)
+
+        exp3 = FakeExperiment([0])
+        exp4 = FakeExperiment([1])
+        par_exp2 = ParallelExperiment([exp3, exp4], flatten_results=True)
+
+        # Set a batch experiment
+        batch_exp = BatchExperiment([par_exp1, par_exp2], flatten_results=True)
+        exp_data = batch_exp.run(backend)
+        
+        for metadata,datum in zip(data,exp_data.data()):
+            self.assertTrue(metadata in datum)
+        
+    
+    def test_new_workflow_child_count(self):
+        
+        backend = FakeBackend()
+
+        exp1 = FakeExperiment([0])
+        exp2 = FakeExperiment([1])
+        par_exp1 = ParallelExperiment([exp1, exp2], flatten_results=True)
+
+        exp3 = FakeExperiment([0])
+        exp4 = FakeExperiment([1])
+        par_exp2 = ParallelExperiment([exp3, exp4], flatten_results=True)
+
+        # Set a batch experiment
+        batch_exp = BatchExperiment([par_exp1, par_exp2], flatten_results=True)
+        exp_data = batch_exp.run(backend)
```

Comment:
```suggestion
        exp_data = batch_exp.run(backend, analysis=None)
        self.assertExperimentDone(exp_data)
```
Let's intentionally remove analysis to make sure the experiment data can bootstrap the child containers.